.rubocop.yml

```diff
@@ -5,7 +5,6 @@ AllCops:
     - "Guardfile"
```

Comment:
- [ ] It would be better if our README file is descriptive of the project. For inspiration, kindly check this [link](https://github.com/microverseinc/readme-template/blob/master/README.md)